src/Content/Post/Entity/PostMedia.php

```diff
@@ -165,7 +165,7 @@ public function getPhotoPath(string $size = ''): string
 	 * Get preview path for given media id relative to the base URL
 	 *
 	 * @param string  $size     One of the Proxy::SIZE_* constants
-	 * @param bool     $vlurred If "true", the preview will be blurred
+	 * @param bool    $blurred  If "true", the preview will be blurred
```

Comment:
While you're at it, can you please rename the parameter to just `$blur`?
```suggestion
	 * @param bool    $blur     If "true", the preview will be blurred
```

Reply:
done.